### PR TITLE
Update README with rnpm linking info

### PR DESCRIPTION
**Pull request by picodoth**
It's so much much much easier to link with rnpm.

### CODE REVIEW

**Comment by yorkie** [comment]
Thanks, may I ask you to add this section not replacing :)

**Comment by picodoth** [comment]
@yorkie I did replace anything, only rename the original 'Link Steps'. rnpm linking and manual linking are different linking methods.

File: README.md [hunk 1]
```diff
@@ -29,7 +29,21 @@ And [react-native-wechat] has the following tracking data in open source world:
 | Dependency  | [![Dependency Status][david-image]][david-url]  |
 | Downloads   | [![Downloads][downloads-image]][downloads-url]  |
 
-## Linking Steps
```
Comment (yorkie) [inline README.md]: Reverse this line

File: README.md [hunk 2]
```diff
@@ -29,7 +29,21 @@ And [react-native-wechat] has the following tracking data in open source world:
...
+## Link with RNPM:
```
Comment (yorkie) [inline README.md]: Add this to the `Linking Steps` section like:

```
### linking with rnpm
```

**Comment by picodoth** [comment]
@yorkie Sry for late response.
I think rnpm is still buggy, I have to add the following steps.

- Only for iOS 9, add `wechat` and `weixin` into `LSApplicationQueriesSchemes` in `Targets` > `info` > `Custom iOS Target Properties`.

- Code the following in `AppDelegate.m` of your project to enable [LinkingIOS]

    ```objective-c
    - (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
    sourceApplication:(NSString *)sourceApplication annotation:(id)annotation
    {
      return [RCTLinkingManager application:application openURL:url
                                sourceApplication:sourceApplication annotation:annotation];
    }
    ```

I would recommend just use the current linking steps.

**Pull request closed without merging**